keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,389 @@
+- [Motivation](#motivation)
+  - [Goals](#goals)
+    - [Enhance determinism](#enhance-determinism)
+    - [Reduce risk](#reduce-risk)
+    - [Collecting data](#collecting-data)
+    - [Creating a policy](#creating-a-policy)
+  - [Non-Goals](#non-goals)
+    - [Long-term support (LTS) releases](#long-term-support-lts-releases)
+    - [Changing enhancements graduation](#changing-enhancements-graduation)
+    - [Architecture changes](#architecture-changes)
+    - [Modifying SIG Architecture policies](#modifying-sig-architecture-policies)
+    - [Accelerated release cycles](#accelerated-release-cycles)
+    - [Establishing maintenance/stability releases](#establishing-maintenancestability-releases)
+    - [Determining an upper bound for Release Team shadows](#determining-an-upper-bound-for-release-team-shadows)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+    - [End User](#end-user)
+    - [Distributors and downstream projects](#distributors-and-downstream-projects)
+    - [Contributors](#contributors)
+    - [SIG Release members](#sig-release-members)
+  - [Risks and Mitigations](#risks-and-mitigations)
+    - [Concentrating risk](#concentrating-risk)
+    - [Attention to tests](#attention-to-tests)
+    - [Attention to dependencies](#attention-to-dependencies)
+    - [Feature graduation](#feature-graduation)
+- [Design Details](#design-details)
+  - [Schedule Policy](#schedule-policy)
+  - [Feedback survey](#feedback-survey)
+- [Implementation History](#implementation-history)
+  - [GitHub Discussion](#github-discussion)
+  - [Leads meeting feedback session](#leads-meeting-feedback-session)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input (including test refactors)
+- [ ] (R) Graduation criteria is in place
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+With this KEP, SIG Release proposes to change the current Kubernetes release
+cadence from 4 down to 3 releases per year. This would start with the 
+Kubernetes 1.22 release cycle.
```

Comment:
Where was the starting release decided? I don't see that in the linked GitHub discussion.

Reply:
Based on the discussions from Leads/TL meeting, sig-release leads assumed this date and the KEP is written to reflect this, with later commits adding more language around this. 

The "decision" would be the KEP merging. At this point, it seems like we need a more formal process for deciding this date/identifying who can be the final source of approval. Is this something steering can assist with?

Reply:
@jeremyrickard wearing my steering hat, sig-arch, sig-release, sig-testing chairs should sign off, please add them as approvers.

( cc @kubernetes/steering-committee in case any other steering folks have other suggestions )

Reply:
Thanks @dims, I will update the approvers section tomorrow.